manifmaker/client/templates/task/TaskListComponent.html

```diff
@@ -4,59 +4,197 @@
+                    <span class="input-group-addon">
+                        <i class="mdi mdi-magnify mdi-inline"></i>
+                    </span>
+                </div>
+            </div>
+
+            <div class="col-md-3 col-md-offset-1">
+                {{> SingleNonMandatorySelectComponent selectLabel="Task responsible" title="Task responsible"
+                optionCollection="Users" optionValueName="name" optionCollectionIndex="UsersIndex"
+                nothingSelectedLabel="No one" quickSelectLabel="select yourself" quickSelectId=currentUserId
+                updateCallback=filterResponsible
+                }}
+            </div>
+
+            <div class="col-md-3 col-md-offset-1">
+                {{> SingleNonMandatorySelectComponent selectLabel="Team" title="Select a team"
+                optionCollection="Teams" optionValueName="name" optionCollectionIndex="TeamsIndex"
+                updateCallback=filterTeam optionQuery=optionQueryTeamsWithoutAlreadyAssigned
+                }}
+            </div>
+
+        </div>
+
+        <div class="row">
+            <div class="col-md-2 col-md-offset-9">
+                <button class="btn btn-primary pull-right" id="advanced-search-button" href="">
+                    {{#if isSearchAdvanced}}
+                        Less <i class="mdi mdi-chevron-up mdi-inline"></i>
+                    {{else}}
+                        More <i class="mdi mdi-chevron-down mdi-inline"></i>
+                    {{/if}}
+                </button>
+                <br><br>
+            </div>
+        </div>
+
+        {{#if isSearchAdvanced}}
+            <div class="row">
+
+                {{#if isInRole RolesEnum.TASKWRITE}}
+                    <div class="col-md-3">
+                        <!-- TODO rendre fonctionnel le custom picker -->
+                        {{> SingleNonMandatorySelectComponent selectLabel="Validation Status"
+                        optionCollection=optionValidationStatus optionValueName="label"
+                        updateCallback=filterValidationStatus
+                        }}
+
+                    </div>
+                {{/if}}
+                <div class="col-md-3 col-md-offset-1">
+                    <label>
+                        <input type="checkbox"> After :
+                    </label>
+                    <!-- TODO rendre fonctionnel le custom picker -->
+                    {{>DateTimePickerComponent date=TODO updateDateCallback=TODO}}
+                    <label>
+                        <input type="checkbox"> Before :
+                    </label>
+                    <!-- TODO rendre fonctionnel le custom picker -->
+                    {{>DateTimePickerComponent date=TODO updateDateCallback=TODO}}
+                </div>
+
+                {{#if isInRole RolesEnum.ASSIGNMENTTASKUSER}}
+                    <div class="col-md-3 col-md-offset-1">
+                        <label>
```

Comment:
@PtiLuky 

Assignment priority is only due to version 1.0, peux tu commenter ce bout de code en laissant un TODO ?